### PR TITLE
Simplified the "Release Process" page

### PR DESCRIPTION
**Pull request by javiereguiluz**
Now that https://symfony.com/roadmap displays a dynamic chart of supported Symfony versions, we must remove the static image displayed in this page. While doing that, I realized that this is a very verbose article ... so I reworded it to explain the same but using half of the original text lines.

### CODE REVIEW

**Comment by xabbuh** [comment]
For the `2.8` branch?

**Comment by javiereguiluz** [comment]
I wondered the same ... but I ultimately chose 3.4 because 2.8 still talks about some 2.x quirks ... so we can leave that for historical reasons.

File: contributing/community/releases.rst [hunk 1]
```diff
@@ -32,113 +40,36 @@ During the development phase, any new feature can be reverted if it won't be
...
-Standard Versions
-~~~~~~~~~~~~~~~~~
-
-A **Standard Minor Version** is maintained for an *eight month* period for bug
-fixes, and for a *fourteen month* period for security issue fixes.
-
-In Symfony 2.x branch, the number of minor versions wasn't constrained, so that
-branch ended up with nine minor versions (from 2.0 to 2.8). Starting from
-3.x branch, the number of minor versions is limited to five (from X.0 to X.4).
+Starting from Symfony 3.x branch, the number of minor versions is limited to
```
Comment (xabbuh) [inline contributing/community/releases.rst]: Starting from the Symfony 3.x branch [...]

File: contributing/community/releases.rst [hunk 2]
```diff
@@ -180,6 +100,8 @@ This release process was adopted to give more *predictability* and
   with Symfony and with projects using Symfony;
 * Give time to the Symfony ecosystem to catch up with the new versions
   (bundle authors, documentation writers, translators, ...).
```
Comment (xabbuh) [inline contributing/community/releases.rst]: should end with a semicolon now

**Review by xabbuh — APPROVED** [review]

**Pull request closed without merging**